### ISSUE TITLE
Specific noise model for Googles quantum computers

### ISSUE THREAD

**Issue by LorcanConlon** [quantumlib/Cirq#3429]
Is there a way to simulate the noise of googles quantum computers using circ? In the same way that any of IBM's computers can be simulated using different backends?
```
backend = provider.get_backend('ibmqx2')
noise_model = NoiseModel.from_backend(backend)
```
Ideally I would want something like above which allows me to easily call the noise properties of for example Bristlecone or Sycamore. 

If this is not possible does anyone know the best way to approach simulating the noise of any of googles quantum computers in qiskit?

I am asking because I wish to compare Google and IBMs quantum computers for implementing a simple circuit.

**Comment by karlunho** [comment]
Thanks LorcanConlon. Simulating noise is something that is very delicate that we want to be thoughtful about. Although you can use noise models in the cirq simulator, we don't yet have a detailed noise model of our processor that we are ready to make public yet. However, you can infer many of the parameters from our quantum supremacy supplemental - https://www.nature.com/articles/s41586-019-1666-5#MOESM1 . Are there specific noise parameters that you want to model to compare these systems ? Perhaps we can help you.

**Comment by LorcanConlon** [comment]
Hi karlunho, thanks for your response. I have read through the supplementary in an attempt to model the noise in this way. I have a simple three qubit circuit which I wish to run. Starting from the IBMQ noise model I have constructed something similar that allows me to investigate the performance of a quantum computer in terms of the parameters IBM use to characterise their devices: thermal relaxation constant T1, dephasing time constant T2, single qubit gate error rate, two qubit gate error rate and readout error. I was hoping to use the same model with parameters from the Google supremacy paper, however there doesn't seem to be a direct equivalence as the two appear to use different noise models (for example Google don't quote a T2 value in their paper). So I guess I was hoping that rather than try to build another noise model there was some simpler way to obtain an equivalent noise model for the google devices?

**Comment by MichaelBroughton** [comment]
@95-martin-orion should we close this now ?

**Comment by 95-martin-orion** [comment]
Yes, with #4666, #5340, and related sub-PRs, this can be closed.

Some additional follow-up work is tracked in #5397.
